main.py

```diff
@@ -17,7 +17,7 @@
 )
```

Comment:
make sure we can still run main:

```
> poetry run python main.py
```

Reply:
i was unable to test this as `poetry install` did not work (i imagine it's because i'm on python 3.11 which is not compatible with certain packages). though i did do some ad-hoc local testing of the stench/breeze logic and it seems right now